docs/source/plotting/MeshPlotHelp.rst

```diff
@@ -20,136 +18,38 @@
-    from mantid.api import AnalysisDataService as ADS
-
-    # load sample shape mesh file for a workspace
-    ws = CreateSampleWorkspace()
-    # alternatively: ws = Load('filepath') or ws = ADS.retrieve('ws')
-    ws = LoadSampleShape(ws, "tube.stl")
-
-    # get shape and mesh vertices
-    sample = ws.sample()
-    shape = sample.getShape()
-    mesh = shape.getMesh()
-
-    # Create 3D Polygon and set facecolor
-    mesh_polygon = Poly3DCollection(mesh, facecolors = ['g'], edgecolors = ['b'], alpha = 0.5, linewidths=0.1)
-
-    fig, axes = plt.subplots(subplot_kw={'projection':'mantid3d'})
-    axes.add_collection3d(mesh_polygon)
-
-    axes.set_mesh_axes_equal(mesh)
-    axes.set_title('Sample Shape: Tube')
-    axes.set_xlabel('X / m')
-    axes.set_ylabel('Y / m')
-    axes.set_zlabel('Z / m')
-
-    plt.show()
-
-
-CSGObject
-#########
-
 For help defining CSG Shapes and Rotations, see :ref:`HowToDefineGeometricShape`.
-Plot a CSGObject defined in an XML string:
-
-.. plot::
-   :include-source:
-
-   # import mantid algorithms, numpy and matplotlib
-   from mantid.simpleapi import *
-   import matplotlib.pyplot as plt
-   import numpy as np
-   from mpl_toolkits.mplot3d.art3d import Poly3DCollection
-
-   ws = CreateSampleWorkspace()
-
-   merge_xml = ' \
-   <cylinder id="stick"> \
-   <centre-of-bottom-base x="-0.5" y="0.0" z="0.0" /> \
-   <axis x="1.0" y="0.0" z="0.0" />  \
-   <radius val="0.05" /> \
-   <height val="1.0" /> \
-   </cylinder> \
-   \
-   <sphere id="some-sphere"> \
-   <centre x="0.7"  y="0.0" z="0.0" /> \
-   <radius val="0.2" /> \
-   </sphere> \
-   \
-   <rotate-all x="90" y="-45" z="0" /> \
-   <algebra val="some-sphere (: stick)" /> \
-   '
-
-   SetSample(ws, Geometry={'Shape': 'CSG', 'Value': merge_xml})
-
-   sample = ws.sample()
-   shape = sample.getShape()
-   mesh = shape.getMesh()
 
-   mesh_polygon = Poly3DCollection(mesh, edgecolors = 'blue', linewidths=0.1)
-   mesh_polygon.set_facecolor((1,0,0,0.5))
+Workbench
+#########
 
-   fig, axes = plt.subplots(subplot_kw={'projection':'mantid3d'})
-   axes.add_collection3d(mesh_polygon)
+To quickly plot all Sample, Container and Component shapes attached to a workspace, right-click on it
+in the Workspaces Toolbox and select ``Show Sample Shape``. A black arrow will be added for the beam direction
+and, if there is a UB matrix, colored arrows for the real (solid line) and reciprocal (dashed line) lattice vectors.
+A sample shape plot also takes into account any goniometer rotations.
 
-   axes.set_mesh_axes_equal(mesh)
-   axes.view_init(elev=10, azim=-150)
+.. image::  ../images/ShowSampleShape.png
+   :align: center
+   :height: 350px
 
-   axes.set_title('Sample Shape: Microphone')
-   axes.set_xlabel('X / m')
-   axes.set_ylabel('Y / m')
-   axes.set_zlabel('Z / m')
 
-   plt.show()
+Scripting
+#########
+Here are some examples of what you can achieve by plotting the sample shapes with a script. Click on ``Source Code``
```

Comment:
Where is the source code link you can click on? Wasn't seeing it in my help page

Reply:
Inside the qthelp docs there is no source code link but if you view the page in a browser then the source code link is just above the plot. What do you think I should do?
![Screenshot from 2022-11-07 14-43-09](https://user-images.githubusercontent.com/55980573/200338288-efb7b0d0-ed3d-45c7-a6a7-9b546532c0b4.png)

Reply:
OK I'd not noticed that difference between the qt and html docs before. I think it's probably OK to just clarify that the source will only be present if the user is viewing the html version of the docs. 

If you don't like that option then you could see if it's possibly to include your own hand made hyperlink to a separate file containing the python code source code. Not sure if that would involve duplicating the python or whether the plot could be generated from the separate file.